eth/filters/statediff.go

```diff
@@ -0,0 +1,114 @@
+package filters
+
+import (
+	"math/big"
+	"reflect"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+// ProcessStateChanges builds the state diff Payload from the modified accounts in the StateChangeEvent
+var emptyPayload Payload
+
+func ProcessStateChanges(event core.StateChangeEvent, crit ethereum.FilterQuery) (Payload, error) {
```

Comment:
A bit torn about if we should use ethereums.FilterQuery- I think that it is intended to be used for logs, but seems to be generic enough that we can use it for state change events too.